hyper/client/aws/ec2.go

```diff
@@ -507,6 +507,31 @@ func StartServer(manifestPath string, remoteCfg HyperConfig.EC2RemoteConfigurati
 	fmt.Println("Project name is:", projectName)
 	client := GetEC2Client(remoteCfg)
 
+	ec2DescribeInput := &ec2.DescribeInstancesInput{
+		Filters: []types.Filter{
+			{
+				Name:   aws.String("instance-state-code"),
+				Values: []string{"16"},
+			},
+		},
+	}
+
+	ec2DescribeResult, err := GetInstances(context.TODO(), client, ec2DescribeInput)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+		return
+	}
+
+	for _, r := range ec2DescribeResult.Reservations {
+		for _, i := range r.Instances {
+			if IsHyperdriveInstance(i) {
```

Comment:
I think this will return true if the user has access to another instance that is running for a different study. We just want to key-in on whether the user already has an instance running for this particular study. 

Lets also move all this code in to its own function, something along the lines of `GetInstanceForStudy(studyName string)` and just call it here since we'll likely want to reuse that in the future and to reduce the responsibility of the `StartServer` function.

Lets make the console line that we output include the information about the size and on how to access the running instance (like the url, don't worry about the ssh command since its probably a lot of work), and also suggest that if the user is wanting to change size they can run `hyper jupyter stop --remote=%s`